src/chains/ethereum/ethereum/tests/api/eth/getProof.test.ts

```diff
@@ -0,0 +1,362 @@
+      // todo: Generate contractCode from GetProof.sol. The generated code is operating system dependent (due to metadata). Differing code generates a different proof.
+      // This should be solved by generating the expected Proof direclty. Once this is done, most of the static assertions can be removed, as well as the deterministic wallet.
+      // See https://github.com/trufflesuite/ganache/issues/3303 for more information.
+
+      beforeEach(async () => {
+        provider = await getProvider({
+          miner: { defaultTransactionGasLimit: 6721975 },
+          wallet: { deterministic: true }
+        });
+        await provider.send("eth_subscribe", ["newHeads"]);
+
+        const accounts = await provider.send("eth_accounts");
+        ownerAddress = accounts[0];
+
+        const deployContractTxHash = await provider.send(
+          "eth_sendTransaction",
+          [
+            {
+              from: ownerAddress,
+              data: contractCode
+            }
+          ]
+        );
+        await provider.once("message");
+
+        const receipt = await provider.send("eth_getTransactionReceipt", [
+          deployContractTxHash
+        ]);
+        ownerBalance = await provider.send("eth_getBalance", [ownerAddress]);
+
+        contractAccount = new Account(Address.from(receipt.contractAddress));
+        const deployedCode = Data.from(
+          await provider.send("eth_getCode", [
+            contractAccount.address.toString()
+          ])
+        );
+
+        const contractHashData = Data.from(keccak(deployedCode.toBuffer()));
+        const contractAccountBalance = await provider.send("eth_getBalance", [
+          contractAccount.address.toString()
+        ]);
+
+        contractAccount.codeHash = contractHashData.toBuffer();
+        contractAccount.nonce = Quantity.from(0x1);
+        contractAccount.balance = Quantity.from(contractAccountBalance);
+        contractAccount.stateRoot = Buffer.from(
+          "fe26ce2b5dafd303bdbd9adbd1bbc1ef59fceea4f81c282158ab624e2fd1787d",
+          "hex"
+        );
+      });
+
+      it("gets the proof for an externally owned account", async () => {
+        const result = await provider.send("eth_getProof", [
+          ownerAddress,
+          [],
+          "latest"
+        ]);
+
+        assert.equal(result.address, ownerAddress, "Unexpected address");
```

Comment:
These tests should probably be using `strictEqual` and `strictDeepEqual`; I don't see any obvious ways there will be a different in results, but it feels weird to be doing "==" comparisons :-)